EmployeeSalary.Api/Models/Entities/ConcreteEmployee/Developer.cs

```diff
@@ -0,0 +1,14 @@
+﻿
+namespace EmployeeSalary.Api.Models.Entities.ConcreteEmployee
+{
+    public class Developer : HasLeaderEmployee<LeadDeveloper>
+    {
+        #region Constructor
```

Comment:
remove region, regions generally are bad practice. also, there's no need for a region for a constructor